<launch>

  <arg name="output" default="screen" />

  <node name="iarrc_float_commander" pkg="iarrc_tutorials" type="iarrc_tutorial_node" output="$(arg output)">
    <param name="float_command_topic" type="string" value="new_float_command" />
    <param name="float_repub_topic"   type="string" value="iarrc/float_repub" />
  </node>

</launch>
